pkg/ebpf/netlink/conntracker.go

```diff
@@ -100,6 +110,28 @@ func NewConntracker(procRoot string, stbSize int) (Conntracker, error) {
 	return ctr, nil
 }
 
+func (ctr *realConntracker) expvarStats() {
+	ticker := time.NewTicker(5 * time.Second)
+	for range ticker.C {
```

Comment:
Hmm, this doesn't look good for the first 5 seconds, expvar is empty. I probably need this: https://github.com/golang/go/issues/17601#issuecomment-311955879